### ISSUE TITLE
NTR: ERCP or MRCP: biliary ducts beaded (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8899]
New term request ERCP or MRCP: biliary ducts beaded (elthat_100428050555): Multiple narrowing areas with widening in between, sclerosing cholangitis included; irregular; Endoscopic retrograde cholangiopancreatography  UMLS: 2021956

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000846
- label: Beaded bile ducts
- parents: [Abnormal biliary tract morphology (HP:0012440)](https://hpo.jax.org/app/browse/term/HP:0012440)
- pmid: PMID:30600377
- def: Multifocal intra- and extra-hepatic bile duct strictures alternating with normal caliber or slightly dilated segments, resembling beads on a string.
- comment: This abnormality can be observed in primary sclerosing cholangitis and can be ascertained by ndoscopic retrograde cholangiopancreatography (ERCP) or MRCP (magnetic resonance cholangiopancreatography).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8899
- synonyms:
